src/main/java/HtmlUtil.java

```diff
@@ -4,47 +4,81 @@
 
 public class HtmlUtil {
 
+    /**
+     * Return a somehow generated html code.
+     *
+     * For example:
+     * <div class="setup">
+     * 	<div style="float: right;" class="meta"><a href="javascript:expandAll();">Expand All</a> | <a href="javascript:collapseAll();">Collapse All</a></div>
+     * 	<a href="javascript:toggleCollapsable('4717033458227362526');">
+     * 		<img src="/files/images/collapsableOpen.gif" class="left" id="img4717033458227362526"/>
+     * 	</a>
+     * &nbsp;<span class="meta">Set Up: <a href="SetUp">.SetUp</a></span>
+     * 	<div class="collapsable" id="4717033458227362526">setup</div>
+     * </div>
+     *
+     * @param pageData ???
+     * @param includeSuiteSetup ???
+     * @return string representing html code
+     * @throws Exception
+     */
     public static String testableHtml(PageData pageData, boolean includeSuiteSetup) throws Exception {
         WikiPage wikiPage = pageData.getWikiPage();
         StringBuffer buffer = new StringBuffer();
-
         if (pageData.hasAttribute("Test")) {
-            if (includeSuiteSetup) {
-                WikiPage suiteSetup = PageCrawlerImpl.getInheritedPage(SuiteResponder.SUITE_SETUP_NAME, wikiPage);
-                if (suiteSetup != null) {
-                    WikiPagePath pagePath = wikiPage.getPageCrawler().getFullPath(suiteSetup);
-                    String pagePathName = PathParser.render(pagePath);
-                    buffer.append("!include -setup .").append(pagePathName).append("\n");
-                }
-            }
-            WikiPage setup = PageCrawlerImpl.getInheritedPage("SetUp", wikiPage);
-            if (setup != null) {
-                WikiPagePath setupPath = wikiPage.getPageCrawler().getFullPath(setup);
-                String setupPathName = PathParser.render(setupPath);
-                buffer.append("!include -setup .").append(setupPathName).append("\n");
+            if (includeSuiteSetup) { // @TODO: Currently untested!
```

Comment:
If `includeSuiteSetup` is `true`, there is actually no test. We changed quite a bit here, so this is potentially dangerous.